application/templates/entity.html

```diff
@@ -265,13 +265,6 @@ <h3 class="govuk-heading-s">{{ type|capitalize }}</h3>
     </script>
   {% endif %}
 
-  <script>
-    const $subNavTabs = document.querySelector('[data-module="dl-subnav"]')
-    const subNavTabsComponent = new DLFrontend
-      .SubNavTabs($subNavTabs)
-      .init({})
-  </script>
-
```

Comment:
document.querySelector('[data-module="dl-subnav"]') finds nothing. so this code emits an error. removing

Reply:
What do you think was supposed to happen? Any history on when the subNavTabs element was removed?

Reply:
so here is the commit that removed the sub nav:
commit: https://github.com/digital-land/digital-land.info/commit/d797a5d1a0a02f2e75a6d7e0c89beef0ace88afd
and here is the PR that it said commit was part of:
PR: https://github.com/digital-land/digital-land.info/pull/48

my guess from the PR is that the entity page just underwent some large changes and its just a bit of code accidently left in from then